src/_listNavigationItem.scss

```diff
@@ -123,12 +124,12 @@ $fd-list-second-level-popover-position: 3.25rem !default;
       .#{$block}__navigation-item-text,
       .#{$block}__navigation-item-icon,
       .#{$block}__navigation-item-arrow {
-        color: var(--sapSelectedColor);
+        color: var(--sapSelectedColor) !important;
```

Comment:
Note: there's already specificity so I couldn't rely on it, had to use !important

Reply:
I wouldn't rely on `important`. Maybe add more level to it to make it prioritized if possible?

Reply:
I too am wondering why not just add another class

Reply:
@N1XUS I wouldn't rely on 4-5 levels of nesting to enforce specificity either :) It becomes a hell if refactoring is needed or if a new functionality is added. 
@mikerodonnell89 `fd-list__navigation-item--indicated` is the additional class added to change the color of the items. We can't have 2 classes doing the same to enforce specificity. 

Anyways, I added additional nesting. It's not a big deal, just something I don't consider a good practice :)